MekHQ/src/mekhq/campaign/Campaign.java

```diff
@@ -7636,7 +7623,7 @@ public String getCombatPersonnelDetails() {
                 countTotal);
         sb.append(buffer);
 
-        for (int i = 0; i <= Person.T_NUM; i++) {
+        for (int i = 0; i < Person.T_NUM; i++) {
```

Comment:
Can you add a comment in Person.java that explains that T_NUM should be increased if you add a role? A comment exists which says where to put new roles, but doesn't tie it into T_NUM.